src/global/event_bus.gd

```diff
@@ -19,4 +19,4 @@ signal exit_minigame
 
 # For debugging
 func _print_all() -> void:
-	print([ui_credits_start, ui_credits_done, exit_minigame])
+	print([game_loaded, ui_credits_start, ui_credits_done, ui_upgrade_bought, exit_minigame])
```

Comment:
```
func _print_all() -> void:
	print("Event bus signals")
	for s in get_script().get_signal_list():
		print(s.name)
```

Proposed alternate implementation that does not rely on people maintaining the method

Reply:
That implementation would re-introduce all the warnings in the editor.

<img width="912" height="547" alt="image" src="https://github.com/user-attachments/assets/a2446f6c-b114-4450-85c3-61c86efc77ab" />

Reply:
Adding the ignore tag to each signal is probably even more annoying to maintain.
The last option is to disable the warning in project.godot as follows:

```
[debug]

gdscript/warnings/unused_signal=0
```

Do you have a preference?

Reply:
I did not realise that was the main reason this method existed. I'm personally ok with suppressing the warning globally, but will say that if I come to the signal bus and see everyone disabling the warning per signal, at least I know what I am supposed to do. It does make that file a lot less readable though

Reply:
https://docs.godotengine.org/en/4.4/classes/class_%40gdscript.html#class-gdscript-annotation-warning-ignore-start

Some googling and this is apparently a thing

Reply:
Yeah we did the annotation in OmniPong and everyone hated it. :laughing: